tests/userTest.js

```diff
@@ -16,7 +16,11 @@ describe('Main: Currently testing user management,', function () {
   } = require('./pactHelper.js')
   const { validAdminAuthHeaders, xmlResponseHeaders, applicationFormUrlEncoded } = require('./pactHelper.js')
 
-  const getUserInformationInteraction = function (provider, requestName, username, responseBody) {
+  const getUserInformationInteraction = async function (provider, requestName, username, responseBody) {
+    if (username === config.testUser) {
+      await provider
+        .given('the user is recreated', { username: username, password: config.testUserPassword })
+    }
```

Comment:
let's check here if its NOT admin, currently we can only use `config.testUser` the given step will not work with any other user